packages/react-component-library/playwright/helpers.ts

```diff
@@ -0,0 +1,8 @@
+import { parseToRgb } from 'polished'
+
+// Note: Importing 'hex-rgb' fails – this is an alternative
```

Comment:
This was a bit annoying – importing `hex-rgb` gives an error about not being able to import ES modules. (There's a PW_EXPERIMENTAL_TS_ESM=1 env var, but it didn't help...)